en/installation/installation-of-a-central-server/using-packages.md

```diff
@@ -72,124 +72,30 @@ systemctl disable firewalld
-Enable PHP 7.3 using the following command:
-```shell
-dnf module enable php:7.3 -y
-```
-
-Check that PHP 7.3 is activated:
-```shell
-dnf module list php
-```
-
-You should have this result:
-```shell
-CentOS Linux 8 - AppStream
-Name                                     Stream                                 Profiles                                                 Summary
-php                                      7.2 [d]                                common [d], devel, minimal                               PHP scripting language
-php                                      7.3 [e]                                common [d], devel, minimal                               PHP scripting language
-php                                      7.4                                    common [d], devel, minimal                               PHP scripting language
-
-Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
-```
-<!--Oracle Linux 8-->
-#### Oracle CodeReady Builder repository
+#### Remi repository
 
-To install Centreon you will need to enable the official Oracle CodeReady
-Builder repository supported by Oracle.
-
-Enable the repository using these commands:
-
-```shell
-dnf -y install dnf-plugins-core oracle-epel-release-el8
-dnf config-manager --set-enabled ol8_codeready_builder
-```
+To install Centreon you will need to install the **remi** repository.
 
-Enable PHP 7.3 using the following command:
-```shell
-dnf module enable php:7.3 -y
-```
+<!--DOCUSAURUS_CODE_TABS-->
+<!--RHEL 8 / CentOS 8 / Oracle Linux 8-->
+Run the following commands:
 
-Check that PHP 7.3 is activated:
 ```shell
-dnf module list php
+dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
+dnf config-manager --set-enabled 'powertools'
 ```
 
-You should have this result:
+Enable PHP 8.0 using the following commands:
 ```shell
-Oracle Linux 8 Application Stream (x86_64)
-Name                                Stream                                 Profiles                                                 Summary
-php                                 7.2 [d]                                common [d], devel, minimal                               PHP scripting language
-php                                 7.3 [e]                                common [d], devel, minimal                               PHP scripting language
-php                                 7.4                                    common [d], devel, minimal                               PHP scripting language
-
-Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
+dnf module reset php
+dnf module install php:remi-8.0
 ```
 <!--CentOS 7-->
-#### Redhat Software Collections repository
-
-To install Centreon you will need to set up the official Software Collections
-repository supported by Redhat. It is required for installing PHP 7 and its associated libraries.
-
-Install the Software Collections repository using this command:
+Run the following commands:
 
 ```shell
-yum install -y centos-release-scl
```

Comment:
If I don't install the software collection then I hit an issue for the package centreon-web that requires httpd24
Once this done, the installation is complete but I cannot start the php service and I see that php73 is present on my platform so what to do ?